# 目标位置的数组

float64[] rad
